quickwit-search/src/search_stream/leaf.rs

```diff
@@ -165,11 +165,17 @@ async fn leaf_search_stream_single_split(
         search_request.end_timestamp,
     );
 
+    let requires_scoring = match &search_request.sort_by_field {
+        Some(field_name) if field_name == "_score" => true,
+        _ => false,
+    };
+
```

Comment:
As far as I understand, there is no way that stream requests have any sort order set, but I still added this in case I'm wrong or something changes in the future.

Reply:
Still relevant. Thanks to Clippy, it is uglier now.